packages/contract/README.md

```diff
@@ -80,51 +87,47 @@ v4.0.2 or higher.
 Let's use `@truffle/contract` with an example contract from [Dapps For Beginners](https://dappsforbeginners.wordpress.com/tutorials/your-first-dapp/). In this case, the abstraction has been saved to a `.sol` file by [@truffle/artifactor](https://github.com/trufflesuite/truffle/tree/master/packages/artifactor):
 
 ```javascript
+const contract = require("@truffle/contract");
 // Require the package that was previosly saved by @truffle/artifactor
-var MetaCoin = require("./path/to/MetaCoin.json");
+const MetacoinArtifact = require("./path/to/MetaCoin.json");
+const instance = contract(MetacoinArtifact);
 
 // Remember to set the Web3 provider (see above).
-MetaCoin.setProvider(provider);
+instance.setProvider(provider);
 
 // In this scenario, two users will send MetaCoin back and forth, showing
 // how @truffle/contract allows for easy control flow.
-var account_one = "5b42bd01ff...";
-var account_two = "e1fd0d4a52...";
+const account_one = "5b42bd01ff...";
```

Comment:
Would you be willing to re-write all of these variable names in camel case? That is the convention followed now in Truffle.